<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Summary</class>
 <widget class="QWidget" name="Summary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>1037</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Summary</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #83A2FF;
	color: #FFFFFF;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
     <property name="leftMargin">
      <number>50</number>
     </property>
     <property name="rightMargin">
      <number>50</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2">
       <item>
        <widget class="QLabel" name="score_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border-radius: 50%; /* Makes the label circular */
border: 2px solid #000000; /* Border color and width */
padding: 10px; /* Adjust padding as needed */
background-color: #83A2FF;
color: #FFD28F;
border: none;
</string>
         </property>
         <property name="text">
          <string>Your Partial Score is:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="leftMargin">
          <number>50</number>
         </property>
         <property name="topMargin">
          <number>50</number>
         </property>
         <property name="rightMargin">
          <number>50</number>
         </property>
         <property name="bottomMargin">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="play_again_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border-radius: 20px; /* Adjust the radius as needed */
	background-color: #FFE3BB;
}
QPushButton:hover {
	background-color: #FFD28F;
	}
QPushButton:pressed {
	background-color: #B4BDFF;
}</string>
           </property>
           <property name="text">
            <string>Play again!</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="quit_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border-radius: 20px; /* Adjust the radius as needed */
	background-color: #FFE3BB;
}
QPushButton:hover {
	background-color: #FFD28F;
	}
QPushButton:pressed {
	background-color: #B4BDFF;
}</string>
           </property>
           <property name="text">
            <string>Quit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="table_layout">
       <item>
        <widget class="QTableView" name="summary_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableView {
	color: white;
	border: 2px solid #000000;
	padding: 10px;
	background-color: #FFE3BB;
	color: black;
	font-weight: bold;
	border: none;
}

QHeaderView::section {
	color: white;
	background-color: #FFD28F;
	border: 1px solid #555;
	height: 36px;
	width: 260px;
	font-weight: bold;
}</string>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
